test/stubs/dss_stub.c

```diff
@@ -0,0 +1,10 @@
+/*
+ * dss_stub.c
+ *
+ * Stub for the dss
+ */
+
+
+void dss() {
+  return;

```

Comment:
You do not need this `return` it is a void function.

Also, I suggest adding a function like: `dss_was_called()` that we can use to make assertions that indeed the `dss()` was entered. This would require some changes.

Reply:
Agreed this need to be done but the tests created do not need this stub to do anything particular. This would be modified by the person that needs this stub